qiskit/algorithms/gradients/__init__.py

```diff
@@ -0,0 +1,74 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+=====================================
+Gradients (:mod:`qiskit.algorithms.gradients`)
+=====================================
+
+.. currentmodule:: qiskit.algorithms.gradients
+
+Base Classes
+============
+
+.. autosummary::
+   :toctree: ../stubs/
+
+   BaseSamplerGradient
+   BaseEstimatorGradient
+
+Gradients
+=========
+
+.. autosummary::
+   :toctree: ../stubs/
+
+   FiniteDiffEstimatorGradient
+   FiniteDiffSamplerGradient
+   LinCombEstimatorGradient
+   LinCombSamplerGradient
+   ParamShiftEstimatorGradient
+   ParamShiftSamplerGradient
+
+Results
+=======
+
+.. autosummary::
+   :toctree: ../stubs/
+
+   EstimatorResult
+   SamplerResult
```

Comment:
Do you intend the following?
```suggestion
   EstimatorGradientResult
   SamplerGradientResult
```